src/App.tsx

```diff
@@ -79,7 +79,10 @@ const Dashboard = () => {
       <Container fluid className="px-0">
         <Row className="g-0">
           <DashboardSidebar />
-          <Col className="main-content vh-100 overflow-auto">
+          <Col
+            className="main-content vh-100 overflow-auto d-flex flex-column"
+            style={{ background: 'rgb(244, 244, 244)' }}
```

Comment:
instead of hardcoding it here we could update boostrap light grey (or grey-100) and then use color classes